types/resolve/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for resolve 1.19.0
+// Type definitions for resolve 1.20.0
 // Project: https://github.com/browserify/resolve
 // Definitions by: Mario Nebl <https://github.com/marionebl>
```

Comment:
```suggestion
// Definitions by: Mario Nebl <https://github.com/marionebl>
// Definitions by: Jordan Harband <https://github.com/ljharb>
```
so i get tagged on types changes